bots/tests-scan

```diff
@@ -97,6 +98,8 @@ sys.dont_write_bytecode = True
 
 from task import github, label, REDHAT_PING
 
+import amqp
```

Comment:
Would be nice to gracefully handle an `ImportError`, so that this can be used on systems without python2-amqp. But this can be a follow-up cleanup.